src/main/java/com/nvidia/spark/ml/linalg/JniRAPIDSML.java

```diff
@@ -65,7 +65,7 @@ public static JniRAPIDSML getInstance() {
 
   public native void dgemm(int transa, int transb, int m, int n, int k, double alpha, double[] A, int lda, double[] B,
                            int ldb, double beta, double[] C, int ldc, int deviceID);
-  public native void dgemm_b(int m, int n, int k, double[] A, double[] B, double[] C, int deviceID);
-
+  public native long dgemm(int transa, int transb, int m, int n, int k, double alpha, double[] A, int lda, long B,
```

Comment:
Sorry, my mistake.  When I suggested for this to be renamed `dgemm` I didn't realize it was a native function where overloading is problematic.  Now I think it's worse than before, because:
- There's no Javadoc for any of this.  How does one know what the parameters mean and why `B` can be a `long` when it represents a matrix?
- Overloaded methods in native code are not so nice, as you discovered with the mangled name required in native code.